sdk/identity/identity/src/credentials/azureCliCredential.ts

```diff
@@ -39,7 +39,8 @@ export const cliCredentialInternals = {
    */
   async getAzureCliAccessToken(
     resource: string,
-    tenantId?: string
+    tenantId?: string,
+    timeout?: number
```

Comment:
Should be consistent that the timing is in MS as we did with the options?

Reply:
These are internally used, so thought of just leaving them as is. I just changed the naming of publicly exposed ones and we are passing the values as is through the functions.